VidLoader/VidLoaderTests/Mocks/MockAVAssetDownloadURLSession.swift

```diff
@@ -7,10 +7,13 @@
 //
 
 import AVFoundation
+import Foundation
 
 final class MockAVAssetDownloadURLSession: AVAssetDownloadURLSession {
 
-    init(noUse: Bool? = nil) {}
+    convenience init(noUse: Bool? = nil) {
+        self.init()
```

Comment:
this line make tests run recursively, so they never stop, but was needed to make it compilable